system/jhtree/jhinplace.cpp

```diff
@@ -567,17 +603,28 @@ void PartialMatch::serialize(MemoryBuffer & out)
         {
             byte offsetBytes = bytesRequired(getMaxOffset());
             byte countBytes = bytesRequired(getCount()-1);
+            byte sequentialFlags = getSequentialOptionFlags();
 
-            byte sizeInfo = offsetBytes;
+            byte sizeInfo = sequentialFlags | offsetBytes;
```

Comment:
What is max value of offsetBytes? Do we know never going to collide with flags?

Reply:
It will never collide.
Offset bytes will be <= 2 for our key files.  Maximum supported value is 7 (to allow 4 byte offsets).